tools/check_install.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+
+# Copyright 2018 Nagoya University (Tomoki Hayashi)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+import logging
+import sys
+
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(levelname)s: %(message)s")
+
+logging.info("python version = " + sys.version)
+
+# check library availableness
+try:
```

Comment:
Is it possible to get the some of packages from `tools/requirements.txt` including the version information (e.g., librosa, chainer with version 4.3.1)? Then, if someone add something in tools/requirements.txt, we may not have to change this script. (but it would be probably difficult).